### PR TITLE
{lib,vis}[GCCcore/11.2.0] gnuplot v5.4.2, libgd v2.3.3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
{vis}[GCCcore/11.2.0] gnuplot v5.4.2, libgd v2.3.3

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in total)
node3100.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a8e0b7099bcb1207190f3cd2a61d5999 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3518.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/03f0700a4fb74ef5617f45bd00d07ce6 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/adbcb62d8f2e10b92c94b03e5c98ed3e for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/621647a0bead953050b76dab64ece97b for a full test report.

File: easybuild/easyconfigs/l/libgd/libgd-2.3.3-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+]
+
+configopts = "--with-fontconfig=$EBROOTFONTCONFIG --with-jpeg=$EBROOTLIBJPEGMINTURBO "
+configopts += "--with-png=$EBROOTLIBPNG --with-zlib=$EBROOTZLIB"
+
+sanity_check_paths = {
+    'files': ['lib/libgd.a', 'lib/libgd.%s' % SHLIB_EXT],
+    'dirs': ['bin', 'include'],
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/l/libgd/libgd-2.3.3-GCCcore-11.2.0.eb:34]: @boegel most libgd tools don't seem to support `--help` or `--version`, but `webpng` does:
```
$ webpng --help
Usage: webpng [-i y|n] [-l] [-t index|none] [-d] [-a] pngname.png
  -i <y|n>   Turns on/off interlace
  -l         Prints the table of color indexes
  -t <index> Set the transparent color to the specified index (0-255 or "none")
  -d         Reports the dimensions and other characteristics of the image
  -a         Prints all alpha channels that are not 100% opaque

If you specify '-' as the input file, stdin/stdout will be used as input/output.
$ echo $?
0
```

Should we add that one?
```suggestion

sanity_check_commands = ['webpng --help']

```

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/69931862fada768a0ed89f5a5b0b0ddb for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Pull request merged**